clarity-ext-scripts/clarity_ext_scripts/covid/rtpcr_analysis_service.py

```diff
@@ -64,18 +64,28 @@ def __init__(self, covid_reporter_key, internal_control_reporter_key):
 
     # TODO Later we might have to account for giving two trails over 38 as positive.
 
-    def _analyze_sample(self, sample):
+    # POS Control
```

Comment:
I think we might want to merge these comments with those above (original)?

Reply:
Yeah, lets write some proper docs for this thing at a later stage.

Reply:
I think it you think it is ok we merge it.